readme.md

```diff
@@ -98,76 +98,152 @@ enum DurationKeys: String, Codable {
 }
 
 extension Defaults.Keys {
-	static let defaultDuration = Defaults.Key<DurationKeys>("defaultDuration", default: .oneHour)
+	static let defaultDuration = Key<DurationKeys>("defaultDuration", default: .oneHour)
 }
 
 defaults[.defaultDuration].rawValue
 //=> "1 Hour"
 ```
 
-
 ### It's just UserDefaults with sugar
 
 This works too:
 
 ```swift
 extension Defaults.Keys {
-	static let isUnicorn = Defaults.Key<Bool>("isUnicorn", default: true)
+	static let isUnicorn = Key<Bool>("isUnicorn", default: true)
 }
 
 UserDefaults.standard[.isUnicorn]
 //=> true
 ```
 
-
 ### Shared UserDefaults
 
 ```swift
+let extensionDefaults = UserDefaults(suiteName: "com.unicorn.app")!
+
 extension Defaults.Keys {
-	static let isUnicorn = Defaults.Key<Bool>("isUnicorn", default: true)
+	static let isUnicorn = Key<Bool>("isUnicorn", default: true, suite: extensionDefaults)
 }
 
-let extensionDefaults = UserDefaults(suiteName: "com.unicorn.app")!
+defaults[.isUnicorn]
+//=> true
+
+// Or
 
 extensionDefaults[.isUnicorn]
 //=> true
 ```
 
+### Use keys directly
+
+You are not required to attach keys to `Defaults.Keys`.
+
+```swift
+let isUnicorn = Defaults.Key<Bool>("isUnicorn", default: true)
+
+defaults[isUnicorn]
+//=> true
+```
+
+### Observe changes to a key
+
+```swift
+extension Defaults.Keys {
+	static let isUnicornMode = Key<Bool>("isUnicornMode", default: false)
+}
+
+let observer = defaults.observe(.isUnicornMode) { change in
+	// Initial event
+	print(change.oldValue)
+	//=> false
+	print(change.newValue)
+	//=> false
+
+	// First actual event
+	print(change.oldValue)
+	//=> false
+	print(change.newValue)
+	//=> true
+}
+
+defaults[.isUnicornMode] = true
+```
+
 
 ## API
 
 ### `let defaults = Defaults()`
 
-#### Defaults.Keys
+#### `Defaults.Keys`
 
 Type: `class`
 
 Stores the keys.
 
-#### Defaults.Key
+#### `Defaults.Key` *(alias `Defaults.Keys.OptionalKey`)*
+
+```swift
+Defaults.Key<T>(_ key: String, default: T, suite: UserDefaults = .standard)
+```
 
 Type: `class`
 
 Create a key with a default value.
 
-#### Defaults.OptionalKey
+The default value is written to the actual `UserDefaults` and can be used elsewhere. For example, with Interface Builder binding.
+
+#### `Defaults.OptionalKey` *(alias `Defaults.Keys.OptionalKey`)*
+
+```swift
+Defaults.OptionalKey<T>(_ key: String, default: T)
+```
 
 Type: `class`
 
 Create a key with an optional value.
 
-##### defaults.clear()
+#### `Defaults#clear`
+
+```swift
+clear(suite: UserDefaults = .standard)
+```
 
 Type: `func`
 
 Clear the user defaults.
 
+#### `Defaults#observe`
+
+```swift
+observe<T: Codable>(
+	_ key: Defaults.Key<T>,
+	options: NSKeyValueObservingOptions = [.initial, .old, .new],
+	handler: @escaping (OptionalKeyChange<T>) -> Void
```

Comment:
Should be KeyChange not OptionalKeyChange

Reply:
I should not copy-paste stuff when I'm sleepy 😆